.travis.yml

```diff
@@ -9,9 +9,4 @@ branches:
 
 install: make deps
 script:
-    - make integ
```

Comment:
Should we kill this option from the Makefile, too? I couldn't find any reference to this name nor any golang magic to do anything different.